regclient/regclient
# Introduce GoReleaser

### Fixes issue

 #280 



### Describe the change

Standardized way of releasing to GitHub
Creates Brew releases automagically

There are some `TODO` items, questions and shabby parts in this PR that need further discussion.





### How to verify it

Not yet verified. I need to be able to execute GitHub Action workflows to verify the solution works. Acceptance criteria for the test would be:
- Artifacts have been published to a Brew tap and is downloadable and usable out of box for Mac users
- The existing Release Format/Artifacts are not affected. The binaries, file names and their contents, Metadata archive (for serving generated SBOMs and signatures), release notes header/body generation remain the same.



### Changelog text

Introduce GoReleaser



### Please verify and check that the pull request fulfills the following requirements
